_pages/learn.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: splash
+permalink: /learn.html
+title: "Python Tutorials & Learning Resources for Scientists"
+classes: flowing
+header:
+  overlay_image: images/header.png
+  overlay_filter: 0.6
+scientists:
+  - image_path: images/pyopensci-people-at-computer.jpg
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "Our catalog of vetted open source tools makes it easier for scientists to find the trusted tools that they need to develop their open science workflows."
+    title: "Help scientists find the open science tools that they need"
+    url: https://www.pyopensci.org/python-packages.html
+    btn_label: View our growing list of accepted scientific Python packages
+    btn_class: btn--primary
+overview:
+  - excerpt: "pyOpenSci creates resources to help you navigate the Python packaging ecosystem with ease. Our materials are community developed and go through extensive technical and pedagogical review."
+development:
+   - image_path: images/pyopensci-learn-community-created.png
+     alt: "alt here "
+     title: "Our Development Process: Community-developed Python Tutorials for Scientists"
+     excerpt: "Our tutorials are created through a multi-stage community review process.
+
+     * Tutorials are developed by the pyOpenSci team or community members.
+
+     * Tutorials are reviewed by tool maintainers to ensure ideas and concepts are accurate.
+
+     * Before publication, tutorials then go through several rounds of community review for accuracy, usability and accessibility."
+whats_next:
+  - image_path: images/pyopensci-learn-github-collaboration.png
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "Through the [Better Software for Science Fellowship](https://bssw.io/pages/bssw-fellowship-program) we will be developing lessons focused on teaching core GitHub collaboration skills needed for both contributing to open source software and for working in an open science team environment.
+
+
+    **Stay up to date with course development and announcements in [our Discourse community](https://pyopensci.discourse.group/), as well as on [Fosstodon](https://fosstodon.org/@pyOpenSci) and [LinkedIn](https://www.linkedin.com/company/pyopensci/)**"
+    title: "What's next: Collaborative GitHub for Scientists"
+toc: false
+---
+
+{% include feature_row id="overview" type="center" %}
+
+[abadlink](https://www.pyopensci.org/badnesshereinlink.html)
```

Comment:
@willingc i intentionally added a bad link here and it didn't catch it in the build. so while this is all super fast, i'm not sure how it's catching bad links or if it is? maybe this is for another day in another pr that just focuses on testing lychee?